src/blocks/gist/edit.js

```diff
@@ -37,6 +37,14 @@ class Edit extends Component {
 		}
 	}
 
+	static __gistCallbackId = 0;
```

Comment:
@olafleur Have you noticed this being used anywhere else? Wondering if it has any purpose of being static or not.

Reply:
Yes, it is in fact useful to make sure every JSONP callback as a unique id (see #1856).

However, it could easily be rewritten as a useState hook if the Edit component was also rewritten with hooks.

I will put back the comment that was explaining it before ;)

Reply:
FWIW, this can be rewritten to use hooks if you'd like. We have been slowly transitioning parts of code to utilize hooks.

Reply:
Yes! I was trying with this PR to see if it was perceived as a good idea. If this get merged I'll gladly convert also the Edit component to Hooks :)